content/6_Advanced/DP_More.mdx

```diff
@@ -21,6 +19,20 @@ frequency: 1
 	<Resource source="GCP" title="15.4.3 - Knuth's Optimization"/>
 </Resources>
 
+Knuth's optimization is, in a way, a special case of Range DP. In general, it is used to solve DP problems with the following transition - $\texttt{dp}[i][j]$ = $\texttt{cost}[i][j] + min_{i≤k<j}(\texttt{dp}[i][k] + \texttt{dp}[k+1][j])$.  
+Further, the cost function must be monotonic on lattice intervals and it must satisfy the quadrangle inequality, i.e., for all $a\leq b\leq c \leq d$:  
```

Comment:
> monotonic on lattice intervals

what

Reply:
I think it's the term the author's of the original paper describing this speedup used to describe the first condition, as indices of an array can only take on integer values. Just the word monotonic should be fine though. I'll resolve it.